features/support/remote_helpers.rb

```diff
@@ -12,13 +12,13 @@ def base_url domain
 
 
 # Returns the remote URL for a new pull request for the given domain and branch
-def pull_request_url domain, branch_name, repo
+def pull_request_url domain, branch, parent_branch, repo
```

Comment:
I was thinking about it, but then didn't feel strong enough to do it. Now that you mention it too, I changed it to named parameters.